### PR TITLE
add node clustering as argva example

### PR DESCRIPTION
**Pull request by gsoosk**
Hi, 
I added an example of node clustering using the ARGVA method. I thought due to [this thread](https://github.com/rusty1s/pytorch_geometric/issues/203) it could be useful.

### CODE REVIEW

**Comment by rusty1s** [comment]
Awesome. Thank you! I will have a closer look and merge ASAP.

**Comment by rusty1s** [comment]
Sorry it took my so long to review this one. I have nothing to complain :) Thank you a lot!

File: examples/argva_node_clustering.py [hunk 1]
```diff
@@ -84,52 +75,53 @@ def train():
     loss = model.recon_loss(z, data.train_pos_edge_index)
```
Comment (Wenjian-Ma) [inline examples/argva_node_clustering.py:75]: missing  loss = loss + model.reg_loss(z) ?

Comment (rusty1s) [reply examples/argva_node_clustering.py:75]: This is added in the line below, right?

Comment (rusty1s) [reply examples/argva_node_clustering.py:75]: This is fixed in master, see https://github.com/pyg-team/pytorch_geometric/blob/master/examples/argva_node_clustering.py#L77.
